lib/Command/UpsertProvider.php

```diff
@@ -44,6 +44,35 @@ class UpsertProvider extends Base {
 	/** @var ICrypto */
 	private $crypto;
 
+	private const EXTRA_OPTIONS = [
+		'unique-uid' => ['shortcut' => null, 'mode' => InputOption::VALUE_OPTIONAL, 'description' => 'Flag if unique user ids shall be used or not. 1 to enable (default), 0 to disable', 'default' => null, 'setting_key' => ProviderService::SETTING_UNIQUE_UID],
```

Comment:
Would prefer to add linebreaks to the nested array to make it a bit more readable ;)